web/templates/dropdown_list.hbs

```diff
@@ -1,18 +1,29 @@
 {{#with item}}
     <li class="list-item" role="presentation" data-unique-id="{{unique_id}}" data-name="{{name}}" tabindex="0">
         {{#if description}}
-        <a class="dropdown-list-item-common-styles">
+        <span class="dropdown-list-item-common-styles">
```

Comment:
The specific bit of refactoring of this from an `a` to a `span` probably belongs in its own commit, since that's the main thing here that could break other functionality. Probably you should just do it in its own commit in the prep PR tweaking the dropdown widget component that I suggested in another comment.

Reply:
Not needed now.

Reply:
Opened #31488 to replace `popper.one` to `popper.on`.